src/docs/product/accounts/pricing.mdx

```diff
@@ -40,9 +40,23 @@ The second is the **On-Demand Period**. The On-Demand Period reflects a one mont
 
 ### Billing Controls
 
-#### On-Demand Spending Cap {#on-demand-cap}
+#### On-Demand Budgets {#on-demand-cap}
 
-You can set a maximum monthly on-demand bill amount by setting an on-demand spending cap. Once you reach the spending cap, you will still be able to access Sentry and prior event data. However, any additional events you send will be rejected.
+You can set a maximum monthly on-demand invoice amount by setting an on-demand budget. Once you reach the maximum budget capacity, you will still be able to access Sentry and prior event data. However, any additional events you send will be rejected.
+
+<Note>
+
+Per-category on-demand budgets are only available on plans with [performance monitoring](/product/performance/) features and if you're in the Early Adopter program.
+If you opted out of the Early Adopter program while on the per-category on-demand budget strategy, you will still retain this on-demand budget until you switch to the shared on-demand budget strategy.
```

Comment:
This might be overkill, but probably best to just explain the whole process.

```suggestion
If you opt out of the Early Adopter program while on the per-category on-demand budget strategy, you'll retain the per-category budgets you've set and the ability to update them. However, if you switch to the shared on-demand budget strategy, you won't be able to switch back unless you opt in to the Early Adopter program again.
```